multiversx_sdk/abi/abi.py

```diff
@@ -316,6 +319,27 @@ def _create_prototype(self, type_formula: TypeFormula) -> Any:
             return CountedVariadicValues([], item_creator=lambda: self._create_prototype(type_parameter))
         if name == "multi":
             return MultiValue([self._create_prototype(type_parameter) for type_parameter in type_formula.type_parameters])
+        if name == "ManagedDecimal":
+            scale = type_formula.type_parameters[0].name
+
+            if scale == "usize":
+                return ManagedDecimalValue(scale=0, is_variable=True)
+            else:
+                return ManagedDecimalValue(scale=int(scale), is_variable=False)
+
+            
```

Comment:
maybe follow the same pattern as for ManagedDecimal

Reply:
indeed, done